lib/serial_terminal.pm

```diff
@@ -17,6 +17,7 @@ use Mojo::Util qw(b64_encode b64_decode sha1_sum trim);
 use Mojo::File 'path';
 use File::Basename;
 use File::Temp 'tempfile';
+use Time::HiRes;
```

Comment:
hm maybe this needs to go here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/cpanfile ?

Reply:
It's a standard library.

Reply:
and already used:
```
os-autoinst-distri-opensuse> grn "use Time::HiRes" | wc -l
27
```